packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -206,7 +218,25 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   const forceShowingCheckPermissions = !minimumFallbackTimerElapsed;
 
   /* @conditional-compile-remove(video-background-effects) */
-  const { toggleVideoEffectsPane } = useVideoEffectsPane(props.updateSidePaneRenderer, mobileView);
+  const [dismissedVideoEffectsError, setDismissedVideoEffectsError] = useState<DismissedError>();
```

Comment:
to avoid this copy pasted between callarrangement and configuration page, could this logic be moved to the `useVideoEffectsPane`?